menas/ui/components/dataset/datasetDetail.controller.js

```diff
@@ -43,7 +43,15 @@ sap.ui.define([
       let cont = new ConformanceRuleDialog(this);
       let view = this.getView();
 
+      // Monitoring
+
+      this.MAX_RUNS = 500;
+      this.byId("dateFromPicker").setDisplayFormat(Formatters.infoDatePattern);
+      this.byId("dateToPicker").setDisplayFormat(Formatters.infoDatePattern);
       this.setDefaultMonitoringDateInterval();
+      this._model.setProperty("/monitoringTotalsVisible", true);
+      this._model.setProperty("/monitoringRecordsVisible", true);
+      this._model.setProperty("/monitoringDetailsVisible", false);
```

Comment:
Now, this might be a stretch but can we cache these settings? If yes, this could go to the new issue.

Reply:
good point, thnx

Reply:
I propose we make this a separate issue

Reply:
ok, agreed

Reply:
#581